.gitignore

```diff
@@ -11,6 +11,7 @@ gomock_reflect_*
 /aro
 /dev-config.yaml
 /e2e.test
+/test/e2e/e2e.test
```

Comment:
Trying to rebase this branch right now. Not sure that I understand why we need this one.

I'll remove it for now to see what is the difference. Probably it is something CI/docker image related and I do not see issues in my locals dev workflow.